.github/actions/constellation_create/action.yml

```
@@ -214,7 +215,13 @@ runs:
             exit 1
         fi
         echo "$(kubectl get nodes -o json | jq '.items | length')/"${NODES_COUNT}" nodes have joined"
-        kubectl wait --for=condition=ready --all nodes --timeout=10m
+        kubectl wait --for=condition=ready --all nodes --timeout=20m
+        kubectlErrorCode=$?
+        if [[ "${kubectlErrorCode}" -ne 0 ]]; then
+          kubectl get pods -n kube-system
+          kubectl get events -n kube-system
+          exit "${kubectlErrorCode}"
```

```suggestion
          echo "::error::Some error description"
          exit "${kubectlErrorCode}"
```
Please write a descriptive error message.